src/Host/Client/Impl/Host/NullRHostConnector.cs

```diff
@@ -2,13 +2,16 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Globalization;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Common.Core;
 
 namespace Microsoft.R.Host.Client.Host {
     internal sealed class NullRHostConnector : IRHostConnector {
-        private static Task<RHost> Result { get; } = TaskUtilities.CreateCanceled<RHost>(new RHostDisconnectedException());
+        private static Task<RHost> Result { get; } = TaskUtilities.CreateCanceled<RHost>(
+            new RHostDisconnectedException(
+                string.Format(CultureInfo.InvariantCulture, Resources.NoConnectionsAvailable, Environment.NewLine, Environment.NewLine)));
```

Comment:
Nope. When submitting test to MessageBox have to do this. `\n` comes as `\n' literal